### PR TITLE
Regression: React + Blaze  reconciliation 

### PR DESCRIPTION
**Pull request by ggazzo**

### CODE REVIEW

File: app/ui-message/client/message.js [hunk 1]
```diff
@@ -551,6 +551,11 @@ const processSequentials = ({ index, currentNode, settings, forceDate, showDateS
 	const previousNode = (index === undefined || index > 0) && getPreviousSentMessage(currentNode);
 	const nextNode = currentNode.nextElementSibling;
 
+	if (!previousNode) {
+		setTimeout(() => {
+			currentNode.dispatchEvent(new CustomEvent('MessageGroup', { bubbles: true }));
```
Comment (tiagoevanp) [inline app/ui-message/client/message.js:556]: I love that <3

File: app/ui-utils/client/lib/RoomManager.js [hunk 2]
```diff
@@ -265,34 +248,29 @@ export const RoomManager = new function() {
...
-					const percent = (100 / totalHeight) * topOffset;
-					const className = [
-						'tick',
-						mentionLink.classList.contains('mention-link--me') && 'tick--me',
-						mentionLink.classList.contains('mention-link--group') && 'tick--group',
-					].filter(Boolean).join(' ');
-					return `<div class="${ className }" style="top: ${ percent }%;"></div>`;
-				})
-				.join('');
+		updateMentionsMarksOfRoom(/* typeName */) {
```
Comment (tiagoevanp) [inline app/ui-utils/client/lib/RoomManager.js:251]: Can you remove that? Or is something important to stay here like a comment?

File: app/ui-utils/client/lib/openRoom.js [hunk 3]
```diff
@@ -61,34 +54,23 @@ export const openRoom = async function(type, name) {
 				return FlowRouter.go('direct', { rid: room._id }, FlowRouter.current().queryParams);
 			}
 
+
```
Comment (tiagoevanp) [inline app/ui-utils/client/lib/openRoom.js:57]: ...

File: app/ui-utils/client/lib/openRoom.js [hunk 4]
```diff
@@ -61,34 +54,23 @@ export const openRoom = async function(type, name) {
...
 			if (room._id === Session.get('openedRoom') && !FlowRouter.getQueryParam('msg')) {
 				return;
 			}
 
-			if (RoomManager.open(type + name).ready() !== true) {
-				if (settings.get('Accounts_AllowAnonymousRead')) {
-					appLayout.render('main');
-				}
+			RoomManager.open(type + name);
```
Comment (tiagoevanp) [inline app/ui-utils/client/lib/openRoom.js:62]: New or not?

File: app/utils/lib/RoomTypesCommon.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import { Meteor } from 'meteor/meteor';
-import { Session } from 'meteor/session';
+// import { Session } from 'meteor/session';
```
Comment (tiagoevanp) [inline app/utils/lib/RoomTypesCommon.js:2]: //

File: app/utils/lib/RoomTypesCommon.js [hunk 6]
```diff
@@ -47,7 +47,7 @@ export class RoomTypesCommon {
 			const routeConfig = {
 				name: roomConfig.route.name,
 				action: roomConfig.route.action,
-				triggersExit: [() => Session.set('openedRoom', '')],
+				// triggersExit: [() => Session.set('openedRoom', '')],
```
Comment (tiagoevanp) [inline app/utils/lib/RoomTypesCommon.js:50]: //

File: app/utils/lib/roomExit.js [hunk 7]
```diff
@@ -1,5 +1,5 @@
 import { Blaze } from 'meteor/blaze';
-import { Session } from 'meteor/session';
+// import { Session } from 'meteor/session';
```
Comment (tiagoevanp) [inline app/utils/lib/roomExit.js:2]: //

File: app/utils/lib/roomExit.js [hunk 8]
```diff
@@ -25,8 +25,8 @@ export const roomExit = function() {
 		}
 		callbacks.run('roomExit');
 
-		Session.set('lastOpenedRoom', Session.get('openedRoom'));
-		Session.set('openedRoom', null);
+		// Session.set('lastOpenedRoom', Session.get('openedRoom'));
```
Comment (tiagoevanp) [inline app/utils/lib/roomExit.js:28]: //

File: client/templates.ts [hunk 9]
```diff
@@ -6,11 +6,22 @@ createTemplateForComponent('MessageActions', () => import('./components/Message/
...
+	renderContainerView: () =>
+		HTML.DIV({
+			style: 'min-height: 36px;',
+		}),
+});
 
 createTemplateForComponent(
 	'DiscussionMetric',
 	() => import('./components/Message/Metrics/Discussion'),
+	{
```
Comment (tiagoevanp) [inline client/templates.ts:19]: Code style consistence... braces in the same line or not?

File: app/ui-utils/client/lib/RoomManager.js (hunk 2 above)
Comment (ggazzo) [reply app/ui-utils/client/lib/RoomManager.js:251]: actually is a feature removed by now, but I want to restore soon as possible

File: app/ui-utils/client/lib/openRoom.js (hunk 4 above)
Comment (ggazzo) [reply app/ui-utils/client/lib/openRoom.js:62]: this is an old method, responsible for fetch the messages, prepare the subscriptions and few other actions. its 100% necessary still

File: client/templates.ts (hunk 9 above)
Comment (ggazzo) [reply client/templates.ts:19]: tell that to my prettier

Comment (tassoevan) [reply client/templates.ts:19]: Yep, under `client/`, Prettier is the ruler.

**Review by tassoevan — DISMISSED** [review]
Respect ESLint.

**Pull request merged**